### PR TITLE
[Feature] Make MultiSelect use InputToggleButton

### PR DESCRIPTION
**Pull request by riitasointi**
## Description

This PR changes the MultiSelect component so that it uses our InputToggleButton component instead of a CSS-based way to display the caret icon.

This adds a new behavior to the component: the popover can now be closed (in addition to opening) by pressing the caret icon. This was not possible previously.

The height of the InputToggleButton was also set to 100% of parent height to avoid "dead zones"

Also fixed a flaky test in SingleSelect. Used waitFor() when determining what happens when the input is blurred. 

## Motivation and Context

We wanted to unify the implementations of caret icons across the different select components. The use of the InputToggleButton was copied from SingleSelect. 

## How Has This Been Tested?

With Chrome & Firefox, Mac VoiceOver in Styleguidist and a CRA app

## Screenshots:

https://user-images.githubusercontent.com/17459942/163395066-6a5e86d4-0edf-4c50-983a-f184f4aadeee.mov

### MultiSelect
* Use internal InputToggleButton component to display the caret icon. 
* This adds a new behavior to MultiSelect: the popover can now be closed (in addition to opening) by pressing the caret icon
* Changed cursor from default to pointer in popover item list

### SingleSelect
* Changed cursor from default to pointer in popover item list

### Dropdown
* Changed cursor from default to pointer in popover item list

### CODE REVIEW

File: src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.baseStyles.tsx [hunk 1]
```diff
@@ -9,19 +9,6 @@ export const baseStyles = (theme: SuomifiTheme) => css`
   &.fi-multiselect {
     & .fi-filter-input_input-element-container {
       position: relative;
-
```
Comment (LJKaski) [inline src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.baseStyles.tsx:12]: Why are these removed?

Comment (LJKaski) [inline src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.baseStyles.tsx:12]: Why are these removed?

**Review by LJKaski — APPROVED** [review]
Looks good to me. Good work 👍

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-ui-components",
-  "version": "7.0.0-beta.1",
+  "version": "7.0.0-beta.2",
```
Comment (ketsappi) [inline package.json:3]: Normally we wouldn't want to change the version during "normal" PRs.
Separate PR for increasing version number.

But this might be just some mishap when merging/rebasing and does not cause any real problem.

**Review by ketsappi — APPROVED** [review]
Seems to work great.

Good job!

**Comment by ketsappi** [comment]
Maybe release notes should be changed a little in the description.

Thinking from the perspective of what the library user would want to read and what is really needed to be told for them.

File: package.json (hunk 2 above)
Comment (riitasointi) [reply package.json:3]: Yes, this is a bit weird since it's a change made in a previous PR, not this one

**Review by LJKaski — APPROVED** [review]
